### DISCUSSION TITLE
How to connect the buttons

### DISCUSSION THREAD

**Discussion by roshin8** [outdoorbits/little-backup-box#236] [Q&A] [answered]
Hi,
I read the wiki on connecting the buttons but i'm still very confused as I have no knowledge on using resistors and stuff. Is there a picture or diagram on how to setup the buttons? Are resistors required? Are they available on Amazon? Does it have to be 10000 ohms?
Really appreciate any help

**Comment by outdoorbits** [comment]
As I found, a resistor isn't strictly needed. But it's cheap and protects your pi. You'll be safe around 10000 ohms. Costs are close to shipping costs, the resistor is nearly for free.
I'll publish a schematic drawing within the next days.

**Comment by outdoorbits** [comment, marked as answer]
The wiki is updated:
https://github.com/outdoorbits/little-backup-box/wiki/02b.-Displaymenu
Is this clear?

**Reply by roshin8** [reply]
Yes! Thank you so much!
